pkg/server/registry/registry.go

```diff
@@ -29,8 +29,10 @@ func APIStores(c kclient.WithWatch, cfg, localCfg *clientgo.Config) (map[string]
 	}
 
 	buildersStorage := builders.NewStorage(c)
-	imagesStorage := images.NewStorage(c)
-
+	imagesStorage, err := images.NewStorage(c, nil)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I am commenting up here because I can't comment on on a hidden line, but didn't we talk about getting rid of the `image/tag` subresource? It's down on line 80 in this file. Should be deleted, as should the corresponding strategy code